modules/ROOT/pages/_partials/block-authors-notes.adoc

```diff
@@ -39,7 +39,7 @@ ifeval::["{do-it}" == "yes"]
 ====
 CONTENT UNDER DEVELOPMENT
 
-This content relates to ICR on pre-2.8 deployments using Inter-Sync Gateway Replication (v1) protocol. It is in the process of being updated to reflect the state in release 2.8.
+This content relates to ICR on pre-2.8 deployments using SG{nbsp}Replicate protocol. It is in the process of being updated to reflect the state in release 2.8.
```

Comment:
Global: Please avoid use of "ICR" (or non-industry acronyms)

Reply:
Global: Not sure if this "Authors notes" is a common thing across all our platforms or industry practice..just seems like the doc is beta quality .Why are we publishing work thats in progress

Reply:
I was used as an include on under-development pages. It's no longer required now and I will remove.